$OpenBSD: patch-src-util_anthy_el,v 1.2 2020/08/24 13:42:26 jca Exp $

- old-style backquotes are obsolete
- last-command-char is obsolete

Index: src-util/anthy.el
--- src-util/anthy.el.orig
+++ src-util/anthy.el
@@ -161,11 +161,11 @@
 
 ;; From skk-macs.el From viper-util.el.  Welcome!
 (defmacro anthy-deflocalvar (var default-value &optional documentation)
-  (` (progn
-       (defvar (, var) (, default-value)
-	 (, (format "%s\n\(buffer local\)" documentation)))
-       (make-variable-buffer-local '(, var))
-       )))
+  `(progn
+     (defvar ,var ,default-value
+       ,(format "%s\n\(buffer local\)" documentation))
+       (make-variable-buffer-local ',var)
+       ))
 
 ;; buffer local variables
 (anthy-deflocalvar anthy-context-id nil "コンテキストのid")
@@ -892,7 +892,7 @@
 	 ((event-matches-key-specifier-p event 'backspace) 8)
 	 (t
 	  (char-to-int (event-to-character event)))))
-    last-command-char))
+    last-command-event))
 
 ;;
 ;;
